src/main/java/modtrekt/logic/commands/RemoveCommand.java

```diff
@@ -4,18 +4,24 @@
 
 import java.util.List;
 
+import com.beust.jcommander.Parameter;
+
 import modtrekt.commons.core.Messages;
 import modtrekt.commons.core.index.Index;
 import modtrekt.logic.commands.exceptions.CommandException;
 import modtrekt.logic.parser.CliSyntax;
+import modtrekt.logic.parser.converters.IndexConverter;
 import modtrekt.model.Model;
 import modtrekt.model.module.Module;
 
 /**
  * Deletes a module identified using it's displayed index from the module list.
  */
 public class RemoveCommand extends Command {
-    public static final String COMMAND_WORD = "remove";
+    public static final String COMMAND_WORD = "remove module";
+    public static final String COMMAND_WORD_SHORTHAND = "remove mod";
+    public static final String COMMAND_WORD_SHORTHAND_2 = "rm module";
+    public static final String COMMAND_WORD_SHORTHAND_3 = "rm mod";
```

Comment:
This can be a `String[]` as the `addCommand` from JCommander accepts an array of strings as aliases.

see #139

Reply:
Great point, thanks! Updated accordingly